service/src/main/java/aero/minova/cas/service/repository/DataEntityRepository.java

```diff
@@ -9,7 +9,7 @@
 import aero.minova.cas.service.model.DataEntity;
 
 @NoRepositoryBean
-public interface DataEntityRepository<E extends DataEntity> extends JpaRepository<E, Long> {
+public interface DataEntityRepository<E extends DataEntity> extends JpaRepository<E, Integer> {
```

Comment:
Warum wurde dies geändert bzw. warum war vorher Long falsch?

Reply:
Im DataEntity ist der KeyLong (also die ID Spalte) als int definiert.

Also habe ich es hier in dem Repository auch zu Integer geändert, damit es konsistent ist, und man bei repository.findById direkt den KeyLong verwenden kann und ihn nicht erst zu einem Long konvertieren muss.

Ob der KeyLong aber nicht insgesamt ein Long sein sollte könnte man sich drüber streiten, da wären aber mehr Anpassungen nötig...